### PR TITLE
Feature/12 timer css refactor

### PR DESCRIPTION
**Pull request by natesheridan**
**What (if any) features are you implementing?**
- Added a timerView to the index.html that displays an active timer including a button to start
    - also added accompanying CSS for that view 
- Added error messages that are hidden by default to each element in the formView
    - also added accompanying CSS for error messages
- 
**What (if anything) did you refactor?**
- Refactored CSS to fit left section padding so it wasn't pushed up against the left edge of the window.
- Refactored styles.css to include error messages, and a new container to hold the view for an active timer.
- Redefined 2 querySelector variables in JS that conflicted after merge.

**Were there any issues that arose?**
- merge had conflicts that needed fixing regarding hiding and viewing different elements.

**Is there anything needed from your teammates?**
- none
**Next intended features**
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,56 +14,72 @@
...
-            <div class="activity-selection">
-              <label for="study">
-              <input type="radio" name="activity_categories" id="study" value="study">
-              <img class="image" src="./assets/study.svg" alt="study button">
-              <p>Study</p>
-              </label>
+      <h1 id="left-section-header">New Activity</h1>
+
+      <!-- Hide this 'formview' with classlist.add 'hidden'  on start activity button event click-->
+      <span class = "" id="formView">
```
Comment (annnuuuh) [inline index.html:20]: remove class assigned to empty string on line 20

**Review by annnuuuh — APPROVED** [review]
I think we just need to delete that empty class on line 20. Otherwise, looks awesome!!

File: index.html [hunk 2]
```diff
@@ -14,56 +14,72 @@
...
+              <div class="time-input-sections">
+                <h2>Minutes</h2>
+                <input class="text-field" type='number' size="50">
+              </div>
+              <div class="time-input-sections">
+                <h2>Seconds</h2>
+                <input class= "text-field" type='number' size="50">
+              </div>
             </div>
+            <p class="error-message">⚠ Please enter a time for the countdown ⚠</p>
```
Comment (pthan1) [inline index.html:63]: Use provided image instead of emoji.

**Review by pthan1 — COMMENTED** [review]
Looks good!

**Pull request merged**